shepherd/LCM.py

```diff
@@ -0,0 +1,49 @@
+import lcm
+import threading
+import json
+
+class LCMClass:
+        def __init__(self, queue, receive_channel):
+            self.queue = queue
+            self.receive_channel = receive_channel
+            self.lc = lcm.LCM()
+
+        # def start_thread(self):
+        #   self.thread = threading.Thread()
+        #   self.thread.daemon = True
+        #   self.thread.start() 
+
+        # Receive messages
+        def lcm_start_read(self):
+            def string_to_int(string):
+                try:
+                    return int(string)
+                except ValueError:
+                    return string
+
+            def handler(channel, item):
+                msg = item.decode()
+                msg_list = msg.split('|')
+                self.queue.put((msg_list[0], [string_to_int(x) for x in msg_list[1:]]))
+
+            self.lc.subscribe(self.receive_channel, handler)
+
+            def run():
+                while True:
+                    self.lc.handle()
+
+            rec_thread = threading.Thread()
+            rec_thread.run = run
+            rec_thread.start()
+
+        # Send a list into a target
```

Comment:
comment style should be three single quotes above and below the docstring and the docstring should be under the def statement and above the rest of the method